modules/services/muchsync.nix

```diff
@@ -0,0 +1,160 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.muchsync;
+in {
+  meta.maintainers = with maintainers; [ pacien ];
+
+  options.services.muchsync = {
+    enable = mkEnableOption "muchsync periodic sync";
+
+    frequency = mkOption {
+      type = types.str;
+      default = "*:0/5";
+      description = ''
+        How often to run muchsync. This
+        value is passed to the systemd timer configuration as the
+        <literal>OnCalendar</literal> option. See
+        <citerefentry>
+          <refentrytitle>systemd.time</refentrytitle>
+          <manvolnum>7</manvolnum>
+        </citerefentry>
+        for more information about the format.
+      '';
+    };
+
+    server = mkOption {
+      type = types.str;
```

Comment:
Perhaps this should be `listOf str` in case somebody would like to synchronize with multiple hosts?

Reply:
Amended to allow multiple remotes.